### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by mana998**
Refactored code

### CODE REVIEW

**Review by dprzygocka — APPROVED** [review]
there is a lot of code missing I hope it won't break stuff

File: public/client.js [hunk 1]
```diff
@@ -1,16 +1,13 @@
+//control variable to check whether game is in-progress
+let playing = false;
```
Comment (dprzygocka) [inline public/client.js:2]: also is this for checking if the game is on to prevent other people from joining?

Comment (mana998) [reply public/client.js:2]: Oh technically that could work like that if it was part of the game.. maybe we can then make it part of the game object? Here it was just to not register onkeyUp and onkeydown when you enter the username

**Comment by mana998** [comment]
The code isnt actually missing, I just moved it to another file so it is easier to find which part of the code we are working on

File: public/client.js (hunk 1 above)
Comment (dprzygocka) [reply public/client.js:2]: yeah that would be a nice idea so I can reuse it

**Pull request merged**